src/InstrumentationEngine/Instruction.cpp

```diff
@@ -1258,26 +1278,34 @@ HRESULT MicrosoftInstrumentationEngine::CSwitchInstruction::GetBranchTarget(_In_
 {
     HRESULT hr = S_OK;
     CLogging::LogMessage(_T("Starting CSwitchInstruction::GetBranchTarget"));
-    *ppTarget = NULL;
 
+    *ppTarget = GetBranchTargetInternal(index);
```

Comment:
Are we using the optimized version anywhere?

Reply:
? It's being used here. I don't understand the questyion.

Reply:
nvm, I see the usage.